catalog/core/catalog-core-solrcommands/src/main/java/org/codice/ddf/commands/solr/SolrCommands.java

```diff
@@ -76,4 +83,32 @@ protected void printErrorMessage(String message) {
         printColor(ERROR_COLOR, message);
     }
 
+    protected void printInfoMessage(String message) {
+        printColor(INFO_COLOR, message);
+    }
+
+    private SolrClient getCloudSolrClient() {
+        String zkHosts = System.getProperty(ZOOKEEPER_HOSTS_PROP);
+        LOGGER.debug("Zookeeper hosts: {}", zkHosts);
+
+        if (zkHosts != null) {
+            SolrClient client = new CloudSolrClient.Builder().withZkHost(zkHosts)
+                    .build();
+            LOGGER.debug("Created solr client: {}", client);
+            return client;
+        } else {
+            return null;
+        }
+    }
+
+    void shutdown(SolrClient client) throws IOException {
+        if(client != null) {
+            LOGGER.debug("Closing solr client");
+            client.close();
+        }
+    }
+
+    SolrClient getSolrClient() {
```

Comment:
❗️If we make this method `protected`, we should either add `@Nullable` on the return type, or change it to throw an exception instead of returning `null`.